### DISCUSSION TITLE
The units in Japanese translation are wrong.

### DISCUSSION THREAD

**Discussion by Mizukaze2303** [breezy-weather/breezy-weather#728] [General]
![Screenshot_20240302-164458.png](https://github.com/breezy-weather/breezy-weather/assets/93418350/580d2e19-0a87-423c-a996-a6018f5babbb)

![Screenshot_20240302-164506.png](https://github.com/breezy-weather/breezy-weather/assets/93418350/b94309f9-4b15-4187-ab3f-7b6600eb2637)

![Screenshot_20240302-164512.png](https://github.com/breezy-weather/breezy-weather/assets/93418350/8c646534-58c0-45c9-981d-23fc6748c488)

![Screenshot_20240302-164519.png](https://github.com/breezy-weather/breezy-weather/assets/93418350/556a5d76-3780-4ba5-bc58-1a07fd23101f)

Maybe they are translated by programs but not been checked, such as _んん(nn)_ should be _mm_ or _ミリ_.

**Comment by papjul** [comment]
All translations happen [on Weblate](https://hosted.weblate.org/projects/breezy-weather/breezy-weather-android/#information). You can discuss with other Japanese translators there when there is a disagreement.

It's forbidden to translate using machine translation tools if you don't check afterwards, but as I don't speak Japanese, I unfortunately cannot make sure all translators respect this rule :(

**Reply by Mizukaze2303** [reply]
Thanks for your guidance and I've already checked and corrected some Japanese translations with problem.
It's my honor that if I can do something for this project.

**Reply by papjul** [reply]
I'm sure Japanese people will appreciate what you did! :) I will merge the changes in next version.

**Reply by Mizukaze2303** [reply]
Well, I corrected some mistakes such as unit and Japanese weather terminology, checked some suggestions and translated some sentences to make them more native.
Best wishes :)
